lib/config/configuration.go

```diff
@@ -1208,17 +1209,33 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *service.Config) error {
 	}
 	ldapPassword, err := os.ReadFile(fc.WindowsDesktop.LDAP.PasswordFile)
 	if err != nil {
-		return trace.WrapWithMessage(err, "loading LDAP password from file %v",
+		return trace.WrapWithMessage(err, "error loading LDAP password from file %v",
 			fc.WindowsDesktop.LDAP.PasswordFile)
 	}
+
+	var cert *x509.Certificate
+	if !fc.WindowsDesktop.LDAP.SkipVerifyCA {
+		raw_cert, err := ioutil.ReadFile(fc.WindowsDesktop.LDAP.CAFile)
+		if err != nil {
+			return trace.WrapWithMessage(err, "error loading LDAP root CA from file %v", fc.WindowsDesktop.LDAP.CAFile)
+		}
+
+		cert, err = x509.ParseCertificate(raw_cert)
```

Comment:
Note that the godoc for `ParseCertificate` says it's possible for both a certificate and an error to be returned, and in that case the error will be a `NonFatalErrors`.

Might be worth investigating under what circumstances we can get a non fatal error, and determine whether we want to proceed in that case.

Reply:
I'm looking at [this godoc here](https://pkg.go.dev/crypto/x509?utm_source=gopls#ParseCertificate) and it doesn't say that. Maybe you're looking at a different library?

Reply:
You're right. Not sure what I was looking at!